src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -11,7 +11,7 @@
  * This class is the glue that binds the controls on the physical operator interface to the commands
  * and command groups that allow control of the robot.
  */
-public class OI {
+public class OI { //Operator Interface
```

Comment:
Comment should be above or below OI and probably not on the same line to follow formatting